.github/workflows/tests-e2e.yml

```diff
@@ -43,7 +43,9 @@ jobs:
         run: pnpm build ${{ matrix.target }} --channel=nightly
 
       - name: Install Playwright Browsers
-        run: pnpm exec playwright install --with-deps
+        run: |
+          pnpm exec playwright install --with-deps
+          pnpm exec playwright install ${{ matrix.project }}
```

Comment:
Here instead of installing all browsers, we are going to install only the one that we are testing for?

Can't we update the command to: `pnpm exec playwright install ${{ matrix.project }} --with-deps`?

Reply:
/me tries.
But I think it'll fail as the setup step uses chromium by default, which needs to be installed too (not just `matrix.project`)

Reply:
`pnpm exec playwright install chromium ${{ matrix.project }} --with-deps` works.